Gemfile

```diff
@@ -7,6 +7,8 @@ gem 'rails', '4.2.6'
 gem 'sqlite3'
 
 gem 'haml-rails'
+gem 'foundation-rails'
+gem 'font-awesome-rails'
```

Comment:
In this branch it's only used for the hamburger button that opens the off-canvas menu.  However, as we build our more of our planned interface, I think we'll find occasion to use even more of its 600+ icons. =)